go/install/install_unix.go

```diff
@@ -26,6 +26,8 @@ import (
 // like Gnome and KDE all support. See
 // http://standards.freedesktop.org/desktop-entry-spec/latest/.
 
+const backtick = "`"
```

Comment:
this is kind of sad. Could not figure out how to do a multiline string with backticks in it...